Solution1/ConsoleApp/Program.cs

```diff
@@ -11,7 +12,16 @@ public class Program
     {
         public static async Task Main()
         {
-            await StartUserCommunication(GetRegistrarDependencies(new FileConfig()));
+            var configRoot = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json")
```

Comment:
can we avoid adding this file? appsettings.json is specific for Web Api, not for Console app